spec/features/game_over_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './app'
+
+feature 'Choose option' do
+  scenario 'choosing rock' do
+    visit('/')
+    fill_in :player_name, with: 'Nora'
+    click_button 'Submit'
+    click_button 'Rock'
+    if page.has_content?("Computer chose Scissors")
```

Comment:
Interesting use of conditionals here! Nice one! 

Another way to make this test deterministic is to make use of a seed to ensure what the computer will choose: 
https://ruby-doc.org/core-2.4.0/Random.html
An example implementation is here: https://github.com/makersacademy/skills-workshops/blob/master/week-3/debugging_2/spec/features/random_happy_emoji_spec.rb#L9